### ISSUE TITLE
Import problem when using with vue 3

### ISSUE THREAD

**Issue by r0hit5292** [kirillmurashov/vue-drag-resize#175]
**Description**
When we are trying to use vue drag and resize with vue3 
basically on importing it gives us error and not working with vue 3
![Screenshot from 2023-01-18 17-48-05](https://user-images.githubusercontent.com/115546545/213377072-bf761ec1-412e-4f79-8fb4-9ee1989364e4.png)

**Comment by RodrigoSpinelli** [comment]
Hello, did you manage to solve this bug?

**Comment by r0hit5292** [comment]
n

> Hello, did you manage to solve this bug?

not yet

**Comment by r0hit5292** [comment]
> Hello, did you manage to solve this bug?

do you have idea how to use it with vue 3

**Comment by RodrigoSpinelli** [comment]
honestly I researched and saw that it is a headache, I will try to make the component by hand

**Comment by r0hit5292** [comment]
yes It is not working properly with Vue3 and also did not get any proper response from vue team also

**Comment by JulessZ** [comment]
Hi, 

same here.
As you mentioned if you split it as an individual component it works fine.

What I did is to copy the folder component to my folder X. And then I imported the component right from there. Not the best way and as it is a component not the right way, but it solved the issue. 

Will see if I can fix the Liebery

**Comment by r0hit5292** [comment]
> Hi,
> 
> same here. As you mentioned if you split it as an individual component it works fine.
> 
> What I did is to copy the folder component to my folder X. And then I imported the component right from there. Not the best way and as it is a component not the right way, but it solved the issue.
> 
> Will see if I can fix the Liebery

have to tried it?

**Comment by disha1202** [comment]
I think this issue is because vue-drag-resize does not have a typescript version

It can be fixed by adding `declare module 'vue-drag-resize';` in .d.ts file.

**Comment by r0hit5292** [comment]
> I think this issue is because vue-drag-resize does not have a typescript version
> 
> It can be fixed by adding `declare module 'vue-drag-resize';` in .d.ts file.

but i guess itsupport type script disha

**Comment by r0hit5292** [comment]
> I think this issue is because vue-drag-resize does not have a typescript version
> 
> It can be fixed by adding `declare module 'vue-drag-resize';` in .d.ts file.
can you explain it disha because you are talking about d.ts file which is present in node modules but what if some copy from my repo and install node modules again

**Comment by disha1202** [comment]
> > I think this issue is because vue-drag-resize does not have a typescript version
> > It can be fixed by adding `declare module 'vue-drag-resize';` in .d.ts file.
> > can you explain it disha because you are talking about d.ts file which is present in node modules but what if some copy from my repo and install node modules again

You'll have to create a .d.ts file somewhere in your project
https://stackoverflow.com/a/49259337/16398346

**Comment by RodrigoSpinelli** [comment]
I managed to solve this error, but now I'm busy, later I can help you if you still need it

**Comment by r0hit5292** [comment]
> I managed to solve this error, but now I'm busy, later I can help you if you still need it
thanks 
can you please explain how you managed to solve this issue?

**Comment by RodrigoSpinelli** [comment]
I managed to solve it this way, I created a root file called tsconfig.json and filled it with this information -> ' { "allowJs": true,
"noImplicitAny":  false} ' and lastly i reinstalled using this command: ' npm i -s vue-drag-resize 'and it worked
![image](https://user-images.githubusercontent.com/86022938/217822498-aada32c6-afc2-4f34-80f7-2f1cd98c0ce7.png)

**Comment by r0hit5292** [comment]
> I managed to solve it this way, I created a root file called tsconfig.json and filled it with this information -> ' { "allowJs": true, "noImplicitAny": false} ' and lastly i reinstalled using this command: ' npm i -s vue-drag-resize 'and it worked ![image](https://user-images.githubusercontent.com/86022938/217822498-aada32c6-afc2-4f34-80f7-2f1cd98c0ce7.png)

but if we off noimpicitAny 
typescript is off in that case we are not any type error also because its mean typescipt is off

**Comment by sparrow522** [comment]
vue3
import VueDragResize from 'vue-drag-resize/src/components/vue-drag-resize.vue';
✌🏻

**Comment by InsHomePgup** [comment]
> vue3 import VueDragResize from 'vue-drag-resize/src/components/vue-drag-resize.vue'; ✌🏻

Thank you ,it's works.
